docs/howto/operate/add-hub.md

```diff
@@ -1,8 +1,67 @@
 # Add a new hub
 
-Adding a new hub requires that we do three things:
+## Step 1: Decide where the hub will live
 
-1. [Create a new GitHub issue for the hub](https://github.com/2i2c-org/pilot-hubs/issues/new).
-2. Create the hub via appending a new hub entry in the appropriate cluster file under
-   `config/hubs` file (see [](../../topic/config.md) for more information).
-3. Start following [team process](team-process) around operating the hubs.
+Hubs need to exist on a kubernetes cluster, so we will need to
+decide which cluster it should live in.
+
+1. For standalone hubs where we are being paid or the client has cloud
+   credits, we will need to create a cluster specifically for this hub
+   before deploying it.
+2. For hubs we are running for free, they will go into the `pilot-hubs`
+   cluster.
+3. Hubs run with the support of [cloudbank](https://www.cloudbank.org/),
+   will go in the `cloudbank` cluster.
+
+Each cluster has configuration + list of hubs it supports under
+`config/hubs/<cluster-name>.cluster.yaml`.
+
+## Step 2: Decide the template used for the hub
+
+Based on the kinda usage expected of the hub select a [hub template](../../topic/hub-templates.md)
```

Comment:
```suggestion
Hub templates map on to different major use-cases that we aim to enable.
Ask the hub user(s) about their needs and expected usage, and use their answer to select a [hub template](../../topic/hub-templates.md).
```

Reply:
Additionally, perhaps we can document a basic list of the current user images and when each should be used? That feels like it could be another PR though...